wandb/sdk/wandb_init.py

```diff
@@ -1365,6 +1365,9 @@ def init(  # noqa: C901
             is preserved to enable future syncing.
         - `"disabled"`: Disables all W&B functionality, making the run’s methods
             no-ops. Typically used in testing to bypass W&B operations.
+        - `"shared"`: Tracks all processes to a single run. This is an
```

Comment:
"Tracks all processes to a single run" doesn't make sense to me. How about "Allows multiple processes, possibly on different machines, to simultaneously log to the same run"?

A user would probably want to know more detail about the constraints and caveats here, like

* Do all workers for a run need to set `mode="shared"`?
* Do all workers need to use the same run settings and config?

I think at least a short warning or recommendation would be useful here, even if we link out to our docs for the full details.